sale_partner_incoterm/models/sale_order.py

```diff
@@ -1,14 +1,24 @@
 # Copyright 2015 Opener B.V. (<https://opener.am>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, models
+from odoo import api, fields, models
 
 
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    incoterm_address_id = fields.Many2one(
```

Comment:
Why not adopting the computed/stored/readonly=False field in order to remove onchanges ?